.github/workflows/pr-title-checker.yml

```diff
@@ -0,0 +1,24 @@
+# Validate that PR titles are prefixed with one of our accepted labels
+name: "PR Title Checker"
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+      - auto_merge_enabled
```

Comment:
Is there a list of possibilities somewhere?

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request